lang/rust/avro/src/encode.rs

```diff
@@ -209,7 +236,8 @@ mod tests {
             &Value::Array(empty),
             &Schema::Array(Box::new(Schema::Int)),
             &mut buf,
-        );
+        )
+        .expect("message should encode");
```

Comment:
let's print value and schema as I suggested in `decode.rs` with the `expect(success())`